src/components/DisplayPages/DisplayPageMain.js

```diff
@@ -25,11 +68,26 @@ const DisplayPageMainStyled = styled.article`
   display: flex;
   flex-direction: column;
 
-  label[for='master-volume'] {
+  /* label[for='master-volume'] {
     span {
       display: inline-block;
       width: 58px;
       text-align: right;
     }
-  }
+  } */
```

Comment:
I'm confused. /*  */usually tells that this part is commented out but it isn't, is it?